virtcontainers/kata_agent.go

```diff
@@ -204,7 +212,7 @@ func (k *kataAgent) createSandbox(sandbox *Sandbox) error {
 			return err
 		}
 	case kataVSOCK:
-		// TODO Add an hypervisor vsock
+		k.Logger().Debug("Using vSock device will be plugged later")
```

Comment:
nop, serial ports don't need a unique context ID, from #383 

>Since we don't know the context IDs that are being used by other processes, we have to check if the context ID we are about to use is free (ioctl VHOST_VSOCK_SET_GUEST_CID)
even implementing above points, we might face race conditions with other processes, because of ioctl VHOST_VSOCK_SET_GUEST_CID doesn't lock the context ID and other process can use it before we launch the QEMU process, to avoid this issue vsocks must be hot plugged

Reply:
sounds good, but not for this PR

Reply:
fair enough!